configure.ac

```diff
@@ -46,10 +46,10 @@ AC_CANONICAL_TARGET
 AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
 AC_PROG_CC
 AC_USE_SYSTEM_EXTENSIONS
-AC_PROG_CC_C99
+AC_PROG_CC
```

Comment:
This change would cause the problem. On CentOS7's gcc, `-std=gnu99` is required.

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../.. -D_GNU_SOURCE -I../../src -DLTFS_CONFIG_FILE=\"/tmp/ltfs/etc/ltfs.conf\" -DLTFS_BASE_DIR=\"/tmp/ltfs\" -DMSG_CHECK -I .. -Wall -Wsign-compare -fsigned-char -D_FILE_OFFSET_BITS=64 -I/usr/include/fuse -I/usr/include/uuid -I/usr/include/libxml2 -D_REENTRANT -I/usr/include -D ENABLE_SNMP -DNETSNMP_ENABLE_IPV6 -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_RPM_4_4_COMPAT -Ulinux -Dlinux=linux -D_REENTRANT -D_GNU_SOURCE -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I/usr/lib64/perl5/CORE -I. -I/usr/include -fkeep-inline-functions -rdynamic -Werror -Wno-deprecated-declarations -g -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -MT libltfs_la-ltfs.lo -MD -MP -MF .deps/libltfs_la-ltfs.Tpo -c ltfs.c  -fPIC -DPIC -o .libs/libltfs_la-ltfs.o
In file included from ../../src/libltfs/ltfstrace.h:72:0,
                 from ltfs.h:101,
                 from fs.h:52,
                 from ltfs.c:60:
../../src/libltfs/ltfs_thread.h:149:70: error: expected ';', ',' or ')' before 'cond'
 static inline int ltfs_thread_cond_init(ltfs_thread_cond_t *restrict cond)
                                                                      ^
../../src/libltfs/ltfs_thread.h:154:75: error: expected ';', ',' or ')' before 'cond'
 static inline int ltfs_thread_cond_timedwait(ltfs_thread_cond_t *restrict cond,
                                                                           ^
../../src/libltfs/ltfs_thread.h:168:70: error: expected ';', ',' or ')' before 'cond'
 static inline int ltfs_thread_cond_wait(ltfs_thread_cond_t *restrict cond,
                                                                      ^
```

Reply:
I removed the commit that removes AC_PROC_CC_99.
Since AC_PROC_CC get precedence, this compiles for me. Is will produce a warning to remove obsolet AC_PROC_CC_99 and suggests to run autoupdate but works out fine.